configurations/micrometer/Readme.md

```diff
@@ -0,0 +1,375 @@
+            public static final String JVM_CLASSLOADER_METRICS_ENABLED = "metrics.jvm.classloader.enabled";
+            public static final String JVM_GC_METRICS_ENABLED = "metrics.jvm.gc.enabled";
+            public static final String JVM_MEM_METRICS_ENABLED = "metrics.jvm.mem.enabled";
+            public static final String JVM_METRICS_ENABLED = "metrics.jvm.enabled";
+            public static final String JVM_PROCESSOR_METRICS_ENABLED = "metrics.jvm.processor.enabled";
+            public static final String JVM_THREAD_METRICS_ENABLED = "metrics.jvm.thread.enabled";
+        
+            @Bean
+            @Primary
+            @Context
+            @Requires(property = JVM_METRICS_ENABLED, value = "true", defaultValue = "true")
+            @Requires(property = JVM_CLASSLOADER_METRICS_ENABLED, value = "true", defaultValue = "true")
+            public ClassLoaderMetrics classLoaderMetrics() {
+                return new ClassLoaderMetrics();
+            }
+        
+            @Bean
+            @Primary
+            @Context
+            @Requires(property = JVM_METRICS_ENABLED, value = "true", defaultValue = "true")
+            @Requires(property = JVM_GC_METRICS_ENABLED, value = "true", defaultValue = "true")
+            public JvmGcMetrics jvmGcMetrics() {
+                return new JvmGcMetrics();
+            }
+        
+            @Bean
+            @Primary
+            @Context
+            @Requires(property = JVM_METRICS_ENABLED, value = "true", defaultValue = "true")
+            @Requires(property = JVM_MEM_METRICS_ENABLED, value = "true", defaultValue = "true")
+            public JvmMemoryMetrics jvmMemoryMetrics() {
+                return new JvmMemoryMetrics();
+            }
+        
+            @Bean
+            @Primary
+            @Context
+            @Requires(property = JVM_METRICS_ENABLED, value = "true", defaultValue = "true")
+            @Requires(property = JVM_PROCESSOR_METRICS_ENABLED, value = "true", defaultValue = "true")
+            public ProcessorMetrics processorMetrics() {
+                return new ProcessorMetrics();
+            }
+        
+            @Bean
+            @Primary
+            @Context
+            @Requires(property = JVM_METRICS_ENABLED, value = "true", defaultValue = "true")
+            @Requires(property = JVM_THREAD_METRICS_ENABLED, value = "true", defaultValue = "true")
+            public JvmThreadMetrics jvmThreadMetrics() {
+                return new JvmThreadMetrics();
+            }
+        }
+        ```        
+
+### Autoconfigured Meter Registries
+
+Metrics autoconfiguration is about configuring and registering one or more meter registries as 
+Micronaut beans. These beans can be injected or pulled from the bean context. They can be
+further configured and are factories for different kinds of meters; i.e. counters, gauges, etc. 
+
+**A quick note about the bean scopes:** The metric-related beans are all declared as @Context 
```

Comment:
This might be required, but in my testing so far, not required on #190.  I will defer to you and the core team as I am not sure what negative implications for early binding this has.